pkg/ride/runtime.go

```diff
@@ -479,14 +472,30 @@ func (a rideAddress) get(prop string) (rideType, error) {
 	}
 }
 
-func (a rideAddress) String() string {
+func makeLinesForAddressBytes(b []byte) []string {
+	r := make([]string, 3)
```

Comment:
In this case arrays are preferable because compiler can check array bounds at compile time. And, btw, I think this function can be simplified to this:
```go
func makeLinesForAddressBytes(b []byte) []string {
	return []string{
		addressTypeName + "(",
		"\t" + bytesField + " = " + rideBytes(b).String(),
		")",
	}
}
```